### ISSUE TITLE
`MountVolume.SetUp failed for volume "lib-modules" : hostPath type check failed: /lib/modules is not a directory`

### ISSUE THREAD

**Issue by scottrobertson** [packethost/csi-packet#82]
Just tried setting this up on an RKE cluster. Getting the following:

```
MountVolume.SetUp failed for volume "lib-modules" : hostPath type check failed: /lib/modules is not a directory
```

On all of the `csi-node-*` pods.

Nodes are running Ubuntu 18.04

**Comment by scottrobertson** [comment]
`kube-apiserver` is running with `allow-privileged`. Cannot run kubelet with that, as it's not a config option since 1.15.

**Comment by scottrobertson** [comment]
The folders do exist on the nodes.

**Comment by displague** [comment]
Thought to be related to this error too:
```
  Warning  FailedMount 8s kubelet, {ip}   MountVolume.MountDevice failed for volume "pvc-{uuid}" :
rpc error: code = Unknown desc = devicePath error, file not found for pattern /dev/disk/by-path/...
```

**Comment by displague** [comment]
Thanks for reporting this issue @scottrobertson.

Unfortunately, this issue will not be fixed - see #103 (Equinix Metal Block Storage from Datera is EOL).
